scripts/exglobal_diag.sh

```diff
@@ -233,8 +233,12 @@ EOFdiag
    rlist="conv_gps conv_ps conv_pw conv_q conv_sst conv_t conv_uv saphir"
    for rtype in $rlist; do
       set +e
-      ${CHGRP_CMD} *${rtype}*
-      ${STRICT_ON:-set -e}
+      for rfile in *"${rtype}"*; do 
+	 if [[ -s "${rfile}" ]]; then
+            ${CHGRP_CMD} "${rfile}"
+	    ${STRICT_ON:-set -e}
+         fi
+      done
```

Comment:
Please use spaces instead of tabs for white space.

```suggestion
      for rfile in *"${rtype}"*; do 
         if [[ -s "${rfile}" ]]; then
            ${CHGRP_CMD} "${rfile}"
            ${STRICT_ON:-set -e}
         fi
      done